src/components/sidebar/helpers/generate-product-landing-nav-items.ts

```diff
@@ -31,6 +29,27 @@ export const generateProductLandingSidebarMenuItems = (
 	}))
 
 	let docsItems
+	const menuItems = []
+	const introNavItem = routes.find((route) => route.fullPath.endsWith('/intro'))
+
+	/**
+	 * This order is meaninful and should align with the global nav order see
+	 * /src/components/navigation-header/components/product-page-content/utils/get-nav-items.ts
+	 *
+	 * - Install
+	 * - Intro (if exists)
+	 * - Tutorials
+	 * - Documentation and all other docs paths (e.g. API / CLI)
+	 * - Integrations
+	 */
```

Comment:
💭 We could improve this order to be drive by global config

Reply:
Added task link in code comment https://app.asana.com/0/1204807665183200/1205002760871766/f